build.gradle

```diff
@@ -36,8 +35,6 @@ allprojects {
     }
 }
 
-apply plugin: 'jacoco'
```

Comment:
we removed this and did not ad anywhere, intentional?

Reply:
It was currently redundant. We already apply it in commons.gradle file